test/test_torch.py

```diff
@@ -16534,26 +16534,27 @@ def test_fn(self, device):
                 break
             t = torch.tensor((1,), device=device, dtype=in_type)
             for out_type in my_float_types:
+                out_t = torch.empty((1,), device=device, dtype=out_type)
                 # Floating type promotion does not support out= calls for input as bool tensors
                 # and output as complex tensors
                 if (in_type == torch.bool and out_type in complex_types):
-                    continue
-                out_t = torch.empty((1,), device=device, dtype=out_type)
-                self.assertEqual(op(t, out=out_t).dtype, out_type)
+                    self.assertRaises(RuntimeError, lambda: op(t, out=out_t))
```

Comment:
Adding `self.assertRaises` is a better strategy for test cases instead of using `continue` and ignoring the calls, IMO.

Reply:
Agreed.